### ISSUE TITLE
İs there any model to apply transfer learning?

### ISSUE THREAD

**Issue by omeryasar** [WongKinYiu/ScaledYOLOv4#269]
Hi, I want to train this repository for my custom objects. But as far I observed, I have to train in scratch in such a case. But rather than that I prefer to fine tune a pretrained model  if its exist. So is there anyone following that approach? if so can you tell me how can i do it. Or is there any pretrained model exist for fine tuning.

Thanks in advance.

**Comment by SashaAlderson** [comment]
These models are trained on coco dataset to detect 80 classes. You can download weights of this models and finetune them to detect your custom objects. There are roboflow guide on youtube for scaledYOLOv4.

**Comment by omeryasar** [comment]
> These models are trained on coco dataset to detect 80 classes. You can download weights of this models and finetune them to detect your custom objects. There are roboflow guide on youtube for scaledYOLOv4.

Hi first thanks for reply. What do you mean by these models?

**Comment by SashaAlderson** [comment]
> > These models are trained on coco dataset to detect 80 classes. You can download weights of this models and finetune them to detect your custom objects. There are roboflow guide on youtube for scaledYOLOv4.
> 
> Hi first thanks for reply. What do you mean by these models?

Download weights from https://github.com/WongKinYiu/ScaledYOLOv4 and you'll get models for finetune. There a guides for that e.g. https://www.youtube.com/watch?v=rEbpKxZbvIo&ab_channel=Roboflow
![image](https://user-images.githubusercontent.com/84590713/123246202-bc946680-d50f-11eb-8c88-a6090cd0d7d7.png)

**Comment by cnnAndBn** [comment]
hi,do you have the backbone weight pretrained in imagenet if I don't choose to use the released model trained in coco?

**Comment by SashaAlderson** [comment]
> hi,do you have the backbone weight pretrained in imagenet if I don't choose to use the released model trained in coco?

No, i don’t have. I believe that this model will do, cause it’s trained to recognize different patterns and got a lot of data(from every image you’re getting a lot of boxes to train on). So i don’t see the problem here with Coco dataset. Correct me if i’m mistaken.
